projects/demo/src/modules/components/primitive-textfield/primitive-textfield.component.ts

```diff
@@ -51,6 +53,11 @@ export class ExampleTuiPrimitiveTextfieldComponent extends AbstractExampleTuiInt
         LESS: example1Less,
     };
 
+    readonly example2: FrontEndExample = {
+        TypeScript: example2Ts,
```

Comment:
use `import` directly, please

Reply:
@mullinsmikey Look for an example I posted in your other PR 😉